app/views/admins/questions/_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= simple_form_for(question, url: url, method: method ) do |f| %>
+  <div>
+    <div class="input-field col s12">
+      <%= f.input :title, class: "validate"%>
+    </div>
+
+    <div class="input-field col s12">
+      <%= f.input :user_email, required: false %>
+    </div>
+
+    <div class="input-field col s12">
+      <%= f.input :user_name %>
+    </div>
+
+    <div class="input-field col s12">
+      <%= f.input :body %>
+    </div>
+  </div>
+
+  <div class="waves-effect waves-light btn">
+    <%= f.submit yield(:button_text), class: "btn-area" %>
+  </div>
+
+  <div class='group-buttons', style="width:100%;">
+    <%= link_to"All questions", admins_questions_path, class: 'article-button outline' %>
```

Comment:
I see the wording is hardcoded. Maybe, better to use I18n?

Reply:
in progress

Reply:
`link_to"All questions"` -> `link_to "All questions"`